### ISSUE TITLE
did not support IE 11

### ISSUE THREAD

**Issue by cooliean** [vue-bulma/vue-admin#178]
did not support IE 11

**Comment by nolan-resu** [comment]
I am having the same issue.

**Comment by mornir** [comment]
From the wiki: 
> for the time being Internet Explorer (10+) is not supported

**Comment by chenkang084** [comment]
yes, i found it doesn't support ie11 .it looks vue-nprogress alert error

**Comment by Sobient** [comment]
Tested: https://admin.vuebulma.com/#/
Result: Blank page
Error: "Object doesn't support attribute or method assign"

Error seems to stem from vue-nprogress.min.js (v0.1.5)
`return t.install=e,t.start=function(){},Object.assign(t.prototype,r,{init:function(r){this.app=r}}`
